go/vt/vttablet/tabletserver/txthrottler/tx_throttler.go

```diff
@@ -167,6 +169,10 @@ type txThrottlerStateImpl struct {
 
 	// tabletTypes stores the tablet types for throttling
 	tabletTypes map[topodatapb.TabletType]bool
+
+	shardMaxLag  int64
+	endChannel   chan bool
```

Comment:
```suggestion
	done   chan bool
```

Reply:
go style guides suggest that including the type in the variable name is redundant.

Reply:
Also `shardMaxLag` -> `maxLag`? Throttler operates on a shard, so it seems implicit.

Reply:
Renamed as suggested.